tests/unittests/tensorsTest.cpp

```diff
@@ -652,6 +652,43 @@ TEST(Tensor, insertWithCountAndAxis) {
   }
 }
 
+TEST(Tensor, zeroQuanitizedTensor) {
+  Tensor Q8T(ElemKind::Int8QTy, {3, 4, 5, 6}, 127, 0);
+  Tensor Q16T(ElemKind::Int16QTy, {3, 4, 5}, 1234.7, 223);
+  Tensor Q32T(ElemKind::Int32QTy, {3, 4}, 500.4, 53452);
+
+  auto Q8H = Q8T.getHandle<int8_t>();
+  EXPECT_TRUE(Q8H.isZero());
+  for (size_t i = 0; i < 3 * 4 * 5 * 6; i++) {
+    EXPECT_EQ(Q8H.raw(i), 0);
```

Comment:
Perhaps you can get the offset from the Tensor's type here and below instead of explicitly writing out the offset? That way it's quite obvious what's going on.

Reply:
Huh -- what's the bug? Can you create an issue to track it?

An alternative could be `const int32_t offsetQ8 = 0;` and so on, and then you can use it everywhere (including in initializing each of the tensors).

Reply:
No bug currently. I just mean hypothetically if there was a bug in getOffset in the future, since we use it both to write the 'zero' into the buffer and to check that the buffer is 'zero', this test wouldn't give any signal that it was wrong. Using a constant gives us a little more coverage.

I like your alternative though, much clearer.